docs/generics.rst

```diff
@@ -250,3 +250,40 @@ want to override; like so::
 You can also use the Django ``method_decorator`` utility to wrap methods that
 have ``message`` as their first positional argument - note that it won't work
 for more high-level methods, like ``WebsocketConsumer.receive``.
+
+
+As route
+--------
+
+Instead of make routes by ``route_class`` you may use ``as_route`` shortcut.
```

Comment:
Instead of making routes using ``route_class`` you may use the ``as_route`` shortcut.